synergy-design-system/synergy-design-system
# chore: 🔧 chromatic failed detecting regressions

# Pull Request

## 📖 Description

During a Shoelace-Update to a new patch version, we introduced [an issue](#675) in our `<syn-alert>`. The issue was found and quickly fixed. However, the original issue went through our visual regression tests unnoticed.

This PR adjusts chromatics threshold parameter to make sure we are catching subtle differences in the future. It is not set to `0.5`. This value was found to be good for the first to when manually testing via chromatics own tools.

It also introduces chromatics new storybook plugin, making chromatic work without CI from the local branch.

### 🎫 Issues

Closes #677 
Closes #607
Closes #371

## 👩‍💻 Reviewer Notes

Please have a look at Chromatic, there should be no diffs there. To test it, you can play around with your local chromatic setup introduced for this branch. Just get in contact to set it up (if needed).

## 📑 Test Plan

- Check out this branch, run `pnpm install && pnpm build` in the root.
- Move to the docs project and start up storybook
- Start storybook and introduce a forced visual regression in one of the tested stories (e.g. the stories labeled "Screenshot").
- Press the "Run tests" button and play around with the "Visual Tests" plane (usually located on the bottom of the story).

## ✅ DoD
